frontend/src/App.tsx

```diff
@@ -14,6 +15,7 @@ function App() {
       <Route path="dao/chain/:chainId/address/:daoId" element={<Dao />}>
         <Route path="projects" element={<Projects />}></Route>
         <Route path="projects/propose" element={<ProjectProposal />}></Route>
+        <Route path="projects/:projectId" element={<ProjectDetails />}></Route>
```

Comment:
I think this route hides the next line and /tribute page is unreachable. Would be good to write some tests for these cases.